.github/workflows/tests.yml

```diff
@@ -15,42 +15,91 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
+                #php-version: ["5.3", "5.4", "5.5", "5.6", "7.0", "7.1", "7.2", "7.3", "7.4", "8.0", "8.1", "8.2"]
```

Comment:
Please add back newer php versions :)

Reply:
Yes, I will, I'm just trying to get things working on as many versions OS and PHP versions. It's an iterative process 😉

Reply:
Very cool ;)